From 9f0e7d4a262b7cc2922d7dcb63f35b5580fa0f90 Mon Sep 17 00:00:00 2001
From: Christophe Marin <christophe@krop.fr>
Date: Wed, 30 Aug 2023 11:43:30 +0200
Subject: [PATCH] Fix detection of yaml-cpp 0.8

yaml-cpp 0.8 changed their CMake target name. As Findyaml-cpp.cmake
also tries to get variables from the pkgconfig file, we don't need
the cmake target.
---
 share/cmake/modules/Findyaml-cpp.cmake | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/share/cmake/modules/Findyaml-cpp.cmake b/share/cmake/modules/Findyaml-cpp.cmake
index 1dd8105..57d5e5c 100644
--- a/share/cmake/modules/Findyaml-cpp.cmake
+++ b/share/cmake/modules/Findyaml-cpp.cmake
@@ -28,11 +28,6 @@
 if(NOT OCIO_INSTALL_EXT_PACKAGES STREQUAL ALL)
     set(_yaml-cpp_REQUIRED_VARS yaml-cpp_LIBRARY)
 
-    if(NOT DEFINED yaml-cpp_ROOT)
-        # Search for yaml-cpp-config.cmake
-        find_package(yaml-cpp ${yaml-cpp_FIND_VERSION} CONFIG QUIET)
-    endif()
-
     if(yaml-cpp_FOUND)
         get_target_property(yaml-cpp_LIBRARY yaml-cpp LOCATION)
     else()
-- 
2.41.0

